topcoffea/params/xsec.yml

```diff
@@ -120,11 +120,34 @@ TWZToLL_thad_Wlept  : 0.003004
 TWZToLL_tlept_Whad  : 0.003004
 TWZToLL_tlept_Wlept : 0.0015
 
-# Number from CMS AN-19-144
-tWll : 0.01123
-
 # Numbers from https://github.com/cmstas/VVVNanoLooper/blob/master/weights/xsec.txt
 VHnobb : 0.952
 SSWW   : 0.02794
 
-data                    : 1.0
+########################
+# Run3 Values (13p6 TeV)
+########################
+
+#Numbers from SMP-24-001
+DYto2L-2Jets_MLL-50_13p6TeV    : 6345.99
+WtoLNu-2Jets_13p6TeV           : 64481.58
+WW_13p6TeV                     : 125.7
+TTto2L2Nu_13p6TeV              : 98.0
+TbarWplusto2L2Nu_13p6TeV               : 4.67
+TWminusto2L2Nu_13p6TeV                 : 4.67
+TTHtoNon2B_13p6TeV                     : 0.24111
+
+#https://twiki.cern.ch/twiki/bin/viewauth/CMS/HiggsZZ4lRunIII
+GluGluToContinto2Zto2E2Mu_13p6TeV      : 0.00624157
+GluGluToContinto2Zto2E2Tau_13p6TeV     : 0.00624157
+GluGluToContinto2Zto2Mu2Tau_13p6TeV    : 0.00624157
+GluGlutoContinto2Zto4Mu_13p6TeV        : 0.00303575
+GluGlutoContinto2Zto4E_13p6TeV         : 0.00305851
+GluGlutoContinto2Zto4Tau_13p6TeV       : 0.00303575
+
+#Numbers from SMP-24-005
+VH_HtoNonbb_13p6TeV                    : 0.9561
+WZZ_13p6TeV                            : 0.06
+ZZZ_13p6TeV                            : 0.02
+
+data                           : 1.0
```

Comment:
Should not need data in more than one place

Reply:
I think I removed it from above and added it at the bottom (just to keep it at the bottom for consistency). there should be only one